### PR TITLE
New 2022 samples

### PR DESCRIPTION
**Pull request by DennRoy**
It took a while, but now most 2022 samples are available in ReReco MiniAODv4 format. Two samples are still stuck in production:

- 2022EE ZZ->2L2Q extension sample
- 2022EE DY->TauTau->MuTau

I can keep an eye on these two samples and uncomment their entries once they're available.

### CODE REVIEW

File: NanoProd/crab/Run3_2022EE/dataFG.yaml [hunk 1]
```diff
@@ -4,16 +4,16 @@ config:
     era: Run3_2022FG
```
Comment (kandrosov) [inline NanoProd/crab/Run3_2022EE/dataFG.yaml:4]: should we change the globaltag?

Comment (DennRoy) [reply NanoProd/crab/Run3_2022EE/dataFG.yaml:4]: My understanding is that the GTs we need, the ones listed [here](https://cms-talk.web.cern.ch/t/mc-offline-rereco-gts-for-2022-remini-nano-v12-in-130x/29645), are the ones that are already defined in RunKit

Comment (kandrosov) [reply NanoProd/crab/Run3_2022EE/dataFG.yaml:4]: hm.. perhaps I'm looking in the wrong place, but those GTs are for 2022FG PromptReco. Weren't there any changes for Mini v4 ReReco?

Comment (DennRoy) [reply NanoProd/crab/Run3_2022EE/dataFG.yaml:4]: I think the previous MiniV3 and NanoV11 GTs were all `124X_...`, while the new ones are `130X_`. And they do call it a "partial ReReco" for data in the link above

Comment (kandrosov) [reply NanoProd/crab/Run3_2022EE/dataFG.yaml:4]: ok, I see. thanks for the explanations! it seems that we are all good with the current setup then.

**Review by kandrosov — CHANGES_REQUESTED** [review]
@DennRoy  could you please also include all the samples requested by Aliaksei in the MM thread? thanks!

**Review by kandrosov — APPROVED** [review]

**Pull request merged**